### ISSUE TITLE
Feature Request : aria-valuetext as formatter value

### ISSUE THREAD

**Issue by mediaformat** [seiyria/bootstrap-slider#641]
Using _formatter_ it is possible to modify the displayed value in the slider tooltip.

This is useful for many cases, such as irregularly spaced ticks, or unevenly stepped values, see :
#535

But also  when the rendered value cannot be accurately represented as a number; Such as week days, month names, etc.

Ex: https://jsfiddle.net/bbw6z63n/3/

Currently the bootstrap-slider is lacking a way to accessibly represent the _formatter_ value to assistive technologies.

**_aria-valuenow_** represents the _value_ of the slider, a helpful addition however would be to add an **_aria-valuetext_** attribute to represent the _formatter value_.

aria-valuetext : https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuetext_attribute

**Comment by rovolution** [comment]
👍

**Comment by rovolution** [comment]
published to 9.5.0. Thanks again for the high-quality PR @mediaformat!

**Comment by mediaformat** [comment]
@rovolution Thanks for your patience
